README.md

```diff
@@ -12,7 +12,7 @@ $ npm install
 ## Requirements
 
 To save the data that is not stored on Ethereum, Bounties requires a running mongodb service. AWS S3 is used to store/serve files.
-The following Environmental variables are required 
+The following Environmental variables are required. An ```.env-example``` file has been included in the repository. Rename this file to ```.env``` and paste the following environmental variables to get started.
```

Comment:
The .env-example is pretty standard in node projects. It's standard/implicit enough that developers will know what that is without us calling it out. If we are going to call it out, please include all of the supported variables and explain what each does and how to set it up and it's significance to the app.